tools/ci/run_ci.sh

```diff
@@ -221,6 +223,11 @@ function run_tests() {
   find "${REPO_ROOT_DIR}" -name \*.gcov -exec rm {} \;
 }
 
+function run_doxygen() {
+  mkdir -p build-wakaama/doxygen_out
+    GIT_REVISION=$(git rev-parse @) WORKING_DIR=$(pwd) DOXYGEN_OUT_DIR=build-wakaama/doxygen_out \
```

Comment:
Would use either a dash, such as in build*-*wakaama:
```suggestion
    GIT_REVISION=$(git rev-parse @) WORKING_DIR=$(pwd) DOXYGEN_OUT_DIR=build-wakaama/doxygen-out \
```

Or just leave out the `_out` part:
```suggestion
    GIT_REVISION=$(git rev-parse @) WORKING_DIR=$(pwd) DOXYGEN_OUT_DIR=build-wakaama/doxygen \
```